nipoppy/workflows/dataset_status.py

```diff
@@ -62,7 +62,36 @@ def run_main(self):
             [status_col_dict["manifest"]] + curation_cols + processing_cols
         ]
 
-        self.logger.debug(status_df)
+        self.logger.info(status_df)
+
+        self.logger.info(f"curation columns: {curation_cols}")
+        self.logger.info(f"processing columns: {processing_cols}")
+
+        # check if col_in_pre_reorg and col_in_post_reorg are all False
+        if (
+            curation_cols
+            and (~status_df[self.curation_status_table.col_in_pre_reorg]).all()
+            and (~status_df[self.curation_status_table.col_in_post_reorg]).all()
```

Comment:
I think this is technically more efficient (and arguably more readable):
```suggestion
            and (not status_df[self.curation_status_table.col_in_pre_reorg].any())
            and (not status_df[self.curation_status_table.col_in_post_reorg].any())
```